### PR TITLE
[docs]: gelato polygon blackberry addition

### PR DESCRIPTION
**Pull request by anirudhmakhana**
This pull request introduces documentation for Polygon Blackberry, a ZK-powered Layer 2 testnet leveraging the Polygon CDK.

Key Additions

- Introduction to Polygon Blackberry: Overview of the testnet’s capabilities and its role in enabling efficient, scalable blockchain applications.
- Gelato Middleware Services: Detailed insights into the middleware services provided by Gelato, including smart contract automation, gasless transactions, and verifiable on-chain randomness.
- Fully-Integrated Partner Services: Information on the extensive range of services from leading web3 infrastructure providers integrated with Blackberry.
- Network Details and Resources: Practical details for developers on how to interact with the Blackberry testnet, including RPC details, faucet, bridge services, and a block explorer link.

Request for Review
I kindly request a review of this addition to ensure it meets the standards of the Polygon repository and accurately represents the Polygon Blackberry testnet. Feedback and suggestions for improvements are highly welcome.

### CODE REVIEW

**Comment by kmurphypolygon** [comment]
@Sharonbc01 is this one good to go?

File: docs/cdk/get-started/connect-testnet.md [hunk 1]
```diff
@@ -11,20 +11,45 @@ The [CDK Stavanger testnet](https://polygon.technology/cdk-stavanger-testnet) is
 - Bridge assets from Sepolia to Stavanger using the bridge.
 - Confirm your transactions with the block explorer.
 
-!!! tips "Setting up your wallet"
-    - Click **Connect wallet** on the [Stavanger explorer](https://sn2-stavanger-blockscout.eu-north-2.gateway.fm/) to auto set up your wallet.
-    - Check out the latest on [setting up a custom network with MetaMask](https://support.metamask.io/hc/en-us/articles/360043227612-How-to-add-a-custom-network-RPC).
+!!! tips "Setting up your wallet" - Click **Connect wallet** on the [Stavanger explorer](https://sn2-stavanger-blockscout.eu-north-2.gateway.fm/) to auto set up your wallet. - Check out the latest on [setting up a custom network with MetaMask](https://support.metamask.io/hc/en-us/articles/360043227612-How-to-add-a-custom-network-RPC).
```
Comment (kmurphypolygon) [inline docs/cdk/get-started/connect-testnet.md]:
```suggestion
!!! tips "Setting up your wallet"
    - Click **Connect wallet** on the [Stavanger explorer](https://sn2-stavanger-blockscout.eu-north-2.gateway.fm/) to auto set up your wallet.
    - Check out the latest on [setting up a custom network with MetaMask](https://support.metamask.io/hc/en-us/articles/360043227612-How-to-add-a-custom-network-RPC).
```

**Comment by Sharonbc01** [comment]
I am not aware of this. Need to check with Gabe in product.

**Comment by kmurphypolygon** [comment]
Do not publish without @gsilva5 OK.

**Comment by anirudhmakhana** [comment]
Hey @kmurphypolygon, Osman recommended us to create a PR here.

**Comment by kmurphypolygon** [comment]
> Hey @kmurphypolygon, Osman recommended us to create a PR here.

Hi, thank you @anirudhmakhana I was not informed about this sorry. I did check with @gsilva5 and he said it's cool but he will give the OK for when to publish.

For info @Sharonbc01

**Comment by kmurphypolygon** [comment]
Converting to draft so no one pushes to live in error.

File: Dockerfile [hunk 2]
```diff
@@ -3,6 +3,7 @@ FROM python:3.9-alpine
 RUN apk update
 RUN apk add rsync
 RUN apk add git
+RUN apk add bash
```
Comment (kmurphypolygon) [inline Dockerfile]:
```suggestion
```

**Comment by kmurphypolygon** [comment]
Hi @anirudhmakhana I have created a new PR #355 to add the details to the doc so I'm closing this one. 

For info, in general we try to avoid marketing-type language in technical documentation and try to align style and structure as much as possible. 

Thank you for your contribution.

**Pull request closed without merging**